test/test_quantization.py

```diff
@@ -311,6 +311,26 @@ def checkQuantized(model):
 
         checkQuantized(model)
 
+    def test_train_save_load_eval(self):
```

Comment:
Should this be called test_post_train_quant_save_load(self):?

Reply:
The class has information about PTQ vs QAT - `EagerModePostTrainingQuantTest`